Sources/PointFree/Mailgun/Mailgun.swift

```diff
@@ -0,0 +1,96 @@
+import Either
+import Foundation
+import HttpPipeline
+import Optics
+import Prelude
+
+enum Tracking: String {
+  case no
+  case yes
+}
+
+enum TrackingClicks: String {
+  case yes
+  case no
+  case htmlOnly = "htmlonly"
+}
+
+enum TrackingOpens: String {
+  case yes
+  case no
+  case htmlOnly = "htmlonly"
+}
+
+public struct Email {
+  var from: String
+  var to: [String]
+  var cc: [String]? = nil
+  var bcc: [String]? = nil
+  var subject: String
+  var text: String?
+  var html: String?
+  var testMode: Bool? = nil
+  var tracking: Tracking? = nil
+  var trackingClicks: TrackingClicks? = nil
+  var trackingOpens: TrackingOpens? = nil
+  var domain: String
+}
+
+public struct SendEmailResponse: Decodable {
+  let id: String
+  let message: String
+}
+
+func mailgunSend(email: Email) -> EitherIO<Prelude.Unit, SendEmailResponse> {
```

Comment:
I'm not capturing the error cause we dont need it right now, but it's basically a simple wrapper around a `message: String` field...